integration-libs/opf/checkout/occ/config/default-occ-opf-config.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OccConfig } from '@spartacus/core';
+
+export const defaultOccOpfConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        getActiveConfigurations: 'active-configurations',
+        initiatePayment: 'payments',
```

Comment:
[QUESTION] (to backend team)

It might just be just me, but I find surprising that very OPF-specific endpoints have such general-sounding names like `/occ/v2/payments` and `/occ/v2/active-configurations` and are the root level `/occ/v2/` of the OCC URL path. I'm wondering why backend developers didn't implement any prefix like `.../opf/...`  or  `.../workbench/...`, for example `/occ/v2/workbench/payments` or `/occ/v2/workbench/active-configuration`?

Reply:
[OUTDATED] We have separate sub-libs "opf-api" for all OPF endpoints.